docs/ViewModel.md

```diff
@@ -244,6 +244,7 @@ reference to the viewModel instance and modify it. This can be seen in the
 following example:
 
 @demo demos/can-component/accordion.html
+@codepen
```

Comment:
All of these should work as of this PR: https://github.com/canjs/canjs/pull/4388